src/app/modules/filters/filters-content/filters-content.component.spec.ts

```diff
@@ -1,25 +1,43 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { Shallow } from 'shallow-render';
 
 import { FiltersContentComponent } from './filters-content.component';
+import { FiltersContentModule } from './filters-content.module';
 
 describe('FiltersContentComponent', () => {
-  // let component: FiltersContentComponent;
-  // let fixture: ComponentFixture<FiltersContentComponent>;
-
-  // beforeEach(async(() => {
-  //   TestBed.configureTestingModule({
-  //     declarations: [ FiltersContentComponent ]
-  //   })
-  //   .compileComponents();
-  // }));
-
-  // beforeEach(() => {
-  //   fixture = TestBed.createComponent(FiltersContentComponent);
-  //   component = fixture.componentInstance;
-  //   fixture.detectChanges();
-  // });
-
-  // it('should create', () => {
-  //   expect(component).toBeTruthy();
-  // });
+  let shallow: Shallow<FiltersContentComponent>;
+
+  beforeEach(() => {
+    shallow = new Shallow(FiltersContentComponent, FiltersContentModule);
+  });
+
+  it('should update the filter object with the passed in filter and value', async () => {
+    const { instance } = await shallow.render({ bind: {} });
+    instance.filters = {};
+
+    instance.updateFilter('value', 'test');
+    expect(instance.filters['test']).toBe('value');
```

Comment:
Linter error, try `instance.filters.test` instead. If that fails then add `// tslint:disable-line:rule-name` on the same line (or `// tslint:disable-next-line:rule-name` on the line before)